tests/test.ts

```diff
@@ -1,147 +1,187 @@
-import util from 'util';
-import test from 'ava';
-import {jsdom} from 'jsdom';
-import m from '.';
+import * as util from 'util';
+import test, {TestContext, Context} from 'ava';
+import * as jsdom from 'jsdom';
+import m from '../src'; // tslint:disable-line:import-name
 
 const isNode8orHigher = Number(process.versions.node.split('.')[0]) >= 8;
 
-const PromiseSubclassFixture = class extends Promise {};
-const ErrorSubclassFixture = class extends Error {};
+/* Currently out of order, see https://github.com/Microsoft/TypeScript/issues/15202
+class PromiseSubclassFixture<T> extends Promise<T> {}*/
+class ErrorSubclassFixture extends Error {}
+
+const document = new jsdom.JSDOM().window.document;
+const createDomElement = (el: string) => document.createElement(el);
+
+interface Test {is(value: any): boolean; fixtures: any[]; }
+const types = new Map<string, Test>([
+	['undefined',
+		{is: m.undefined, fixtures: [undefined]}
+	], ['null',
+		{is: m.null_, fixtures: [null]}
```

Comment:
Why not `m.null`? Isn't the whole purpose of Lines `225-229` in the source file to allow that?

Reply:
It does work when using it in js but Typescript can't find a type definition for that due to it being added via defineProperties 😞 
I know it is not optimal, do you have a better idea or a method that merges the namespace with reserved keywords here? 
Unfortunately `@ts-ignore` did not work out in that case.